python-package/lightgbm/dask.py

```diff
@@ -422,7 +422,7 @@ def _train(
 
     # Tell each worker to train on the parts that it has locally
     #
-    # This code treates ``_train_part()`` calls as not "pure" because:
+    # This code creates ``_train_part()`` calls as not "pure" because:
```

Comment:
thank you! Actually, I meant to write "treats". Like "it treats the calls as not pure"

```suggestion
    # This code treats ``_train_part()`` calls as not "pure" because:
```